corsafe-poc-service/src/main/java/com/aus/corsafe/config/WebclientConfig.java

```diff
@@ -0,0 +1,38 @@
+package com.aus.corsafe.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.web.reactive.function.client.WebClient;
+
+@Configuration
+public class WebclientConfig {
+
+
+    //api tokenSaaS
+    @Bean
+    public WebClient webClient() {
+        return WebClient.builder()
+                .baseUrl("https://jfk-1.tasklist.camunda.io/c4765f54-b30c-4eba-b09f-2914741db450/v1")
+                .defaultHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
+                .defaultHeader(HttpHeaders.AUTHORIZATION, "Bearer " + "eyJhbGciOiJSUzI1NiIsIn" +
+                        "R5cCI6IkpXVCIsImtpZCI6IlFVVXdPVFpDUTBVM01qZEVRME0wTkRFelJrUkJORFk0T0RZeE1FRT" +
+                        "BSa1pFUlVWRVF6bERNZyJ9.eyJodHRwczovL2NhbXVuZGEuY29tL2NsdXN0ZXJJZCI6ImM0NzY1Zj" +
+                        "U0LWIzMGMtNGViYS1iMDlmLTI5MTQ3NDFkYjQ1MCIsImh0dHBzOi8vY2FtdW5kYS5jb20vb3JnSW" +
+                        "QiOiJlMDAzODY2Mi0yNmI2LTQyNTgtOWFjMy01N2IxNTY3YmJkYWMiLCJodHRwczovL2NhbXVuZ" +
+                        "GEuY29tL2NsaWVudElkIjoiN0prZWVDcDRsOTNlOE0uSW1iMUVnYTBVc1pnMVlscHciLCJpc3" +
+                        "MiOiJodHRwczovL3dlYmxvZ2luLmNsb3VkLmNhbXVuZGEuaW8vIiwic3ViIjoibXdvOTB0MnI" +
+                        "zMTYwN3ozNkJOSDY5dFdGS0JYNTVqMVdAY2xpZW50cyIsImF1ZCI6InRhc2tsaXN0LmNhbXV" +
```

Comment:
Create one class as AplicationConfig and Keep all static content in config class